input/pagecontent/general-requirements.md

```diff
@@ -89,13 +89,13 @@ How the system processes the resource depends on local requirements that could a
 
 When a *Must Support* element requires a more tightly stated obligation, this obligation is specified in the AU Core Requester [obligation extension](https://hl7.org/fhir/extensions/StructureDefinition-obligation.html) on the element definition.
 
-#### Presentation of elements labelled Must Support in profiles
+#### Presentation of Must Support and obligations in profiles
 
-##### Presentation of elements labelled Must Support in table views
+##### Presentation of Must Support and obligations in table views
 
 When rendered in an implementation guide each profile is presented in different formal views under tabs labelled "Differential Table", "Key Elements Table", and "Snapshot Table".
 
-The elements labelled *Must Support* in these views are flagged with an <span style="padding-left: 3px; padding-right: 3px; color: white; background-color: red" title="This element must be supported">S</span>. Implementers should refer to the "Key Elements Table" to see the full set of elements that are mandatory or *Must Support*, and the full set of terminology requirements.
+The elements labelled *Must Support* in these views are flagged with an <span style="padding-left: 3px; padding-right: 3px; color: white; background-color: red" title="This element must be supported">S</span>.  The elements with defined obligations in these views are flagged with an <span style="padding-left: 3px; padding-right: 3px; color: white; background-color: red" title="This element has obligations">O</span>. Implementers should refer to the "Key Elements Table" to see the full set of elements that are mandatory or *Must Support*, and the full set of terminology requirements.
```

Comment:
Replace "The elements with defined obligations" to be "The elements labelled with Obligation in these views are"

Then go on to rephrase the rest of this section now that it is about Must Support and Obligation e.g. "Implementers should refer to the "Key Elements Table" to see the full set of elements that are mandatory or *Must Support* or with Obligations, and the full set of terminology requirements."